tests/integration/components/basic-test.js

```diff
@@ -0,0 +1,103 @@
+import { module, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+
+import generateTable, { generateColumns, generateRows } from '../../helpers/generate-table';
+import { componentModule } from '../../helpers/module';
+
+import { find, scrollTo } from 'ember-native-dom-helpers';
+
+import TablePage from 'ember-table/test-support/pages/ember-table';
+import { collection, hasClass } from 'ember-classy-page-object';
+
+let table = TablePage.extend({
+  body: {
+    rows: collection({
+      isCustomRow: hasClass('custom-row')
+    })
+  }
+}).create();
+
+module('Integration | basic', function() {
+
+  componentModule('rendering', function() {
+
+    test('it renders', async function(assert) {
+      let rowCount = 10;
+      let columnCount = 10;
+
+      await generateTable(this, { columnCount, rowCount });
+
+      // Check column header count
+      assert.equal(table.headers.length, columnCount, 'renders the correct number of columns');
+      assert.equal(table.rows.length, rowCount, 'renders the correct number of rows');
+    });
+
+    test('occlusion works', async function(assert) {
+      let rowCount = 100;
+      let columnCount = 10;
+
+      await generateTable(this, { columnCount, rowCount });
+
+      assert.ok(table.rows.length < rowCount, 'not all rows have been rendered');
+      assert.equal(table.getCell(0, 0).text.trim(), '0A', 'correct first row rendered');
+      assert.notEqual(
+        table.getCell(table.rows.length - 1, 0).text.trim(), '99A', 'correct last row rendered'
```

Comment:
I think the message here does not reflect what it test. It should say "Last rendered row is not last data row"